python/tvm/auto_scheduler/cost_model/xgb_model.py

```diff
@@ -141,6 +146,12 @@ def update(self, inputs, results):
         self.inputs.extend(inputs)
         self.results.extend(results)
 
+        if len(self.inputs) - self.last_train_length < self.last_train_length / 5:
```

Comment:
```suggestion
        if len(inputs) < self.last_train_length / 5:
```
Could you explain a bit more on this logic or make it more straightforward?

Reply:
When there're too many logs, auto scheduler will spent much time on cost model training than program measuring in each search round.
This modification is just used to reduce the times of cost model training. At the beginning, we train the cost model in each search round, and when many logs are accumulated, we train the cost model after several search rounds.